### ISSUE TITLE
Halloween Event Basic Reward

### ISSUE THREAD

**Issue by Lithenir** [WarEmu/WarBugs#18010]
**Expected behavior and actual behavior:**
Just grabbed the first Influence Reward Fiendfang from the Event. 
It is supposed to give you a buff for 25% Renown and Exp. I directly used it and the Buff just gives me 10% on Renown and Exp.

**Steps to reproduce the problem:**
Grab Fiendfang and use it

**Testing Screenshots/Videos/Evidences (always needed):**
![WAR-64 2020-10-31 15-56-47](https://user-images.githubusercontent.com/66906405/97782655-17e5a300-1b93-11eb-8f1d-6c3353aac71a.png)
![WAR-64 2020-10-31 15-56-50](https://user-images.githubusercontent.com/66906405/97782663-1ae09380-1b93-11eb-918e-8d28b84bae76.png)

**Comment by dalen** [comment]
It gives 25%, waiting on a client file update.

**Comment by Lithenir** [comment]
Thanks for the fast reply and fix but there seems to be another problem. At least if I understand it right.
The Buff should last for 3 in-game days, meaning the time should be consumed only when I am online with this char, right?

Well, shortly after I took the first two screenshots I went offline. Buff lasts for 72h. Now after around 3 hours I logged back in the game just to see that the Buff lasts for 70h now. So while I was offline the duration shortended by 2 hours.
![WAR-64 2020-10-31 18-47-15](https://user-images.githubusercontent.com/66906405/97786159-052a9880-1baa-11eb-8fc4-60ef97f58106.png)
